### DISCUSSION TITLE
Skin fails to import

### DISCUSSION THREAD

**Discussion by insecurestudios** [ppy/osu#26078] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Attempting to import a skin does not work. I am on the linux version via flatpak on chromebook..

### Screenshots or videos

![Screenshot 2023-12-22 6 53 10 PM](https://github.com/ppy/osu/assets/126529308/e3d0bfc3-f64b-4205-8fb8-399ff65a2911)

### Version

2023.1223.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13757475/compressed-logs.zip)

**Comment by camellirite** [comment]
the flatpak version isn't the official. i'd download the appimage from the releases page here and install it if you want support from the devs. looks like you can install appimages on chromebooks https://beebom.com/install-appimage-programs-on-chromebook/

**Comment by peppy** [comment]
```
2023-12-23 00:52:44 [error]: Could not import (OSU! Lazer.osk)
2023-12-23 00:52:44 [error]: SharpCompress.Common.InvalidFormatException: /mnt/chromeos/MyFiles/Downloads/OSU! Lazer.osk is not a valid archive
```

The skin isn't a valid skin. Could you please upload it here as an attachment for further investigation? Probably just a corrupt download though.

**Comment by insecurestudios** [comment]
[OSU! Lazer.zip](https://github.com/ppy/osu/files/13759454/OSU.Lazer.zip)

**Comment by insecurestudios** [comment, marked as answer]
After trying some more things, the issue was that I did not properly share it with the osu file, rather it was just shared with Linux.
